docs/examples/md-translation.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Convert a Markdown Document to multiple Languages
+nav_order: 11
+parent: Examples
+---
+
+# Introduction
+
+Today we will be creating a workflow that takes a string of markdown and convert it to Spanish and German. It will showcase how to use a foreach state to run the workflow.
```

Comment:
Suggestion:
`takes a string of markdown and convert it`
could be
`takes a markdown document as a string and converts it`